### ISSUE TITLE
format_value value rounding down due to floating points

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/formatters#118]
> format_value(0.63125, format = "xx.xx%")
[1] "63.12%"

> format_value(0.6375, format = "xx.x%")
[1] "63.7%"

**Comment by gmbecker** [comment]
This is correct behavior under the IEC  60559 standard which calls for rounding to the nearest *even* number due to the vagaries of binary floating point precision.

That said, after implementing #336 the default behavior will not change but users will be able to get the rounding-to-nearest-value behavior you were probably expecting.

In my opinion the IEC is still the correct behavior but rounding differences can be a sticking point for companies migrating from SAS to the NEST/rtables stack

**Comment by shajoezhu** [comment]
hi @gmbecker , I was wondeirng if you could add these to the sas rounding test cases. Thanks
